wolox-books/src/app/services/user.service.ts

```diff
@@ -7,11 +7,18 @@ import { environment } from 'environments/environment';
 })
 export class UserService {
 
-  readonly REGISTER_URL = `${environment.booksApiUrl}/users`;
+  readonly API_URLS = {
```

Comment:
The way we usually use the base URLs is just to set the base and the endpoints are in each method. For example,

```ts
readonly BASE_URL = `${environment.booksApiUrl}/users`;
...
loginUser(userData) {
  return this.http.post(`${this.BASE_URL}/sign_in`, userData, { observe: 'response' });
}
```